src/cim/components/InfraEnv/InfraEnvFormPage.tsx

```diff
@@ -124,6 +125,13 @@ const InfraEnvForm: React.FC<InfraEnvFormProps> = ({ onValuesChanged }) => {
                     This will determine for the infrastructure environment which kind of hosts would
                     be able to be added. If the hosts that you want to add are using DHCP server,
                     select this option, else, select the static IP.
+                    {!isBMPlatform && (
+                      // TODO(mlibra): This limitation needs to be updated once https://github.com/openshift/enhancements/pull/871 lands.
+                      <>
+                        <br />
+                        The static IPs are supported on bare metal platform only.
```

Comment:
@matthewcarleton Can you please have a look at phrasing?

Reply:
How about this?
`Static IPs are only supported on bare metal platforms.`